postgres.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM postgres
+
+RUN apt-get update
```

Comment:
These `RUN` statements could, and maybe should, be combined into a single command. Each statement in a Dockerfile produces a new image layer. Unless you have a specific requirement to have the separate layers, then you're better off combining the `RUN` statements so that all commands run but only produce a single layer.

e.g.
```
RUN apt-get update \
  && apt-get install -y git-core \
  && git clone --depth=1 https://github.com/alphagov/verify-event-system-database-scripts.git db-repo \
  && cp db-repo/migrations/*.sql /docker-entrypoint-initdb.d/
```

Reply:
Done as a new commit

Reply:
The problem with doing that, though, is that Docker can't cache the intermediary layers if something changes in a later one. Having them separate means that the calls to apt may not actually need to run, right? Whereas, since the repo will always be different when it's triggered by merges, that will force the other commands to always get run too?

Reply:
I agree with Alex here, combining these would lead to Docker having to run all of these commands every time we build an image, due to layer cache miss.

Reply:
You could split it though:
```
RUN apt-get update \
  && apt-get install -y git-core
RUN git clone --depth=1 https://github.com/alphagov/verify-event-system-database-scripts.git db-repo \
  && cp db-repo/migrations/*.sql /docker-entrypoint-initdb.d/
```
Cache the base `apt-get` layers and have the version specific stuff `git` as a separate layer!?

Reply:
Two layers FTW. Pushed the change.